doc/source/rllib-algorithms.rst

```diff
@@ -406,6 +407,26 @@ HalfCheetah    13000       ~15000
    :start-after: __sphinx_doc_begin__
    :end-before: __sphinx_doc_end__
 
+.. _maml:
+
+Model-Agnostic Meta-Learning (MAML)
+-----------------------------------
+|pytorch| |tensorflow|
+`[paper] <https://arxiv.org/abs/1703.03400>`__ `[implementation] <https://github.com/ray-project/ray/blob/master/rllib/agents/maml/maml.py>`__
+
+RLlib's MAML implementation is a meta-learning method for learning and quick adaptation across different tasks for continuous control. Code here is adapted from https://github.com/jonasrothfuss, which outperforms vanilla MAML and avoids computation of the higher order gradients during the meta-update step. MAML is evaluated on custom environments that are described in greater detail `here <rllib-examples.html#custom-envs-and-models>`__.
```

Comment:
The link is not very useful, can you link to specific files here as examples of the custom env? Also, make sure the files have good pydoc.

Reply:
Aight, should the documentation in `rllib-examples.rst` be deleted in that case?

Reply:
Done